0-ad-history-mod-package/simulation/templates/structures/han/temple.xml

```diff
@@ -10,6 +10,12 @@
   <Identity>
     <Civ>han</Civ>
     <SpecificName>Sìmiào</SpecificName>
+    <History>
+        The Han Empire's vast lands was a place of cultural diversity, and home to many beliefs and religions. The most widespread ones were Taoism and Buddhism. However, the central imperial government followed a legalistic and confucianistic ideology. Legalism claims the necessity of law, justice, and control, while Confucianism is all about morality, reigning gently and obeying.
+        The Han political system was complex but clearly structured. There was a very strict hierarchy, and great respect was demanded for higher-ranking people. Han emperors were, unlike many other emperors in history, not worshipped as gods but seen as Mandates of Heaven, representing the gods on earth.
```

Comment:
```suggestion
        The Han political system was complex but clearly structured. There was a very strict hierarchy, and great respect was demanded for higher-ranking people. Han emperors were, unlike many other emperors in history, not worshiped as gods but seen as Mandates of Heaven, representing the gods on earth.
```
You used the British spelling for worship

Reply:
Yeah, my bad